src/test/MarkdownViewer.test.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { mount } from "enzyme";
```

Comment:
I did one of these are proof of how to migrate off of Enzyme (since it's [dead](https://stackoverflow.com/questions/72029189/does-enzyme-support-react-version-18))